### PR TITLE
TimeInterval to Measurement initializers and vice versa

### PR DESCRIPTION
**Pull request by bobgilmore**
In order to make time-based Measurement objects easier to
use, add initializers to convert between TimeIntervals and
Measurements.

This allows us to create Measurment objects that specify
their own units, pass them around in our code, and then
easily create TimeIntervals when needed to pass to APIs
that require them.

### CODE REVIEW

**Review by paulrolfe — APPROVED** [review]
Bump the podspec to 9.0.2, please. 🚀 💥

**Comment by IntrepidBuildServer** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">The iPhone simluator specified in your Jenkinsfile is invalid for this version of Xcode</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 169 tests, with 0 failures (0 unexpected) in 0.557 (0.647) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by IntrepidBuildServer** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 169 tests, with 0 failures (0 unexpected) in 0.496 (0.636) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Review by MaxHasADHD — APPROVED** [review]

File: Intrepid.podspec [hunk 1]
```diff
@@ -18,7 +18,8 @@ Pod::Spec.new do |s|
                       "Alan Scarpa" => "alan@intrepid.io",
                       "Paul Rolfe" => "paul@intrepid.io",
                       "Colin Tan" => "ctan@intrepid.io",
-                      "Stephen Wingchi Wong" => "stephenwong@intrepid.io"
+                      "Stephen Wingchi Wong" => "stephenwong@intrepid.io",
+                      "Bob Gilmore" => "bgilmore@intrepod.io"
```
Comment (alexpersian) [inline Intrepid.podspec]: `@intrepid.io` here

**Review by alexpersian — APPROVED** [review]

**Comment by IntrepidBuildServer** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 169 tests, with 0 failures (0 unexpected) in 0.454 (0.532) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by ghost** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 169 tests, with 0 failures (0 unexpected) in 0.547 (0.644) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Pull request merged**